samples/Paging/Program.cs

```diff
@@ -79,6 +108,32 @@ async Task BasicScrollingAsync()
     Console.WriteLine($"Total Charge {totalCharge} RU's");
 }
 
+
+async Task BasicSpecificationScrollingAsync()
+{
+    double totalCharge = 0;
+
+    ContinuationTokenSpecificationImplementation specification = new(null,pageSize: 25);
```

Comment:
It would be good to see examples using the new base types implemented inside the library.